src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -672,11 +673,15 @@ def create_python_udf_or_sp(
         else ""
     )
 
+    strict_as_sql = "\nSTRICT" if strict_as_sql else ""
+    if strict:
+        strict_as_sql = "\nSTRICT"
```

Comment:
Sorry, I probably didn't do the suggestion properly. Could you remove line 677-678 please?

Reply:
Oh yeah let me do that lol